runtime/tests/jniarg/args_10.c

```diff
@@ -0,0 +1,207 @@
+
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "jniargtests.h"
+
+jboolean JNICALL Java_JniArgTests_nativeIZrZ(JNIEnv *p_env, jobject p_this, jint value, jbooleanArray backChannel)
+{
+	J9JavaVM *javaVM = getJ9JavaVM(p_env);
+	jboolean converted = (jboolean)value;
+	jboolean elements[] = {JNI_FALSE};
+	PORT_ACCESS_FROM_JAVAVM(javaVM);
+
+	jniTests++;
+
+	if (converted) {
+		// if C could would see this as true, set the backchannel to true
```

Comment:
Some of our compilers historically haven't supported `//` comments yet.  Can you change this to `/* */` in C files?

Reply:
Already updated with `/* */` and will double-check test cases with `-Xjit:count=0`.